velox/exec/tests/GroupedExecutionTest.cpp

```diff
@@ -0,0 +1,523 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <velox/type/Timestamp.h>
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/exec/PartitionedOutputBufferManager.h"
+#include "velox/exec/TableScan.h"
+#include "velox/exec/tests/utils/Cursor.h"
+#include "velox/exec/tests/utils/HiveConnectorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/type/Type.h"
+
+using namespace facebook::velox;
```

Comment:
For the new tests, the guidelines are now to put the test into anonymous namespace inside the namespace of the code being tested.

```
namespace facebook::velox::exec {

namespace {

class GroupedExecutionTest
```

Reply:
Why anonymous namespace?
Why not
`namespace facebook::velox::exec::test {`
?

That way don't need to add
```
using namespace facebook::velox::exec::test;
```

Reply:
> Why anonymous namespace?

This is what we decided upon. I'd imagine we would use xxx::test namespace for test utilities shared across multiple tests, but not for the tests themselves.

CC: @Yuhta @pedroerp